native/packages/react-native-bpk-component-text/src/BpkText.js

```diff
@@ -46,14 +46,28 @@ const styleForTextStyle = (textStyle) => {
     }
   };
 
+  const determineIosFontFamily = (fontWeight) => {
+    let fontName = '.SFUIDisplay';
+    if (Platform.Version >= 11) {
+      fontName = '.SFProDisplay';
+    }
+    switch (fontWeight) {
+      case '700':
```

Comment:
Is this needed? From my tests the platform correctly handles weights on its own?